### ISSUE TITLE
Naming of cityGMLBase.xsd 

### ISSUE THREAD

**Issue by TatjanaKutzner** [opengeospatial/CityGML-3.0Encodings#52]
Is there a reason, why the XSD file of the `Core` module has been named `cityGMLBase.xsd` in CityGML 2.0? 

I would find it more intuitive to name the file `core.xsd` in CityGML 3.0. Also all other XSDs are named according to the module name.

**Comment by TatjanaKutzner** [comment]
In the web meeting on 8 July we discussed that it would be fine to rename the xsd `cityGMLBase.xsd` to `core.xsd`.

In addition, we also discussed that it will be useful to provide a profile schema file that imports all CityGML modules. This profile file already exists for CityGML 2.0 (http://schemas.opengis.net/citygml/profiles/base/2.0/CityGML.xsd) and should also be provided for CityGML 3.0.
